### PR TITLE
ments solver

### PR DESCRIPTION
**Pull request by sone2000**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -159,4 +159,5 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-.vscode/*
\ No newline at end of file
+.vscode/*
```
Comment (larkz) [inline .gitignore:162]: @sone2000, please do not include these .vscode files in the PR, they are IDE dependent.

**Comment by larkz** [comment]
In general, please ensure all the pip pacakges are installed so we can reproduce this.

File: samples/gym/cartPole/CartPole.py [hunk 2]
```diff
@@ -96,8 +96,8 @@ def evaluate_solver(solver_class, iterations_list, goals, trials=10):
     iterations_list = [1, 2, 3, 5, 10, 50, 100, 500, 1000, 1500, 2000, 2500]
     goals = [10, 20, 50, 100, 150, 200, 250, 300]
```
Comment (larkz) [inline samples/gym/cartPole/CartPole.py:98]: Can you explain what these `iterations_list` and `goals` are?

Comment (sone2000) [reply samples/gym/cartPole/CartPole.py:98]: The evaluation loops through the `goals` and `iterations_list` to find the minimum number of MCTS iterations required to achieve each goal. A goal represents the target number of steps the CartPole must stay balanced. By iterating over the `iterations_list`, the evaluation determines the smallest number of MCTS iterations needed for the solver to consistently keep the CartPole balanced for the specified goal steps.

File: Readme.md [hunk 3]
```diff
@@ -22,6 +22,7 @@ python3 -m pip install --index-url https://test.pypi.org/simple/ mcts4py
 Install Requirements:
 
 `python3 -m pip install -r requirements.txt`
+`python3 -m pip install -r requirements_gym.txt`
```
[the remark this answers is not in the document]
Comment (sone2000) [reply Readme.md:25]: I don't really find way to directly set a flag in python requirements. There are two ways I found to combine two requirements.txt file, the first one requires to make changes in setup.py, the second one is to grep the package name and install it

[the remark this answers is not in the document]
Comment (larkz) [reply Readme.md:25]: can you implement it via the setup.py, and update the README.md file how to do it please.

**Pull request merged**